### PR TITLE
SDL2: update to 2.0.16.

### PR DESCRIPTION
**Pull request by ardadem**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ardadem** [comment]
testing locally, please do not merge yet

**Comment by ardadem** [comment]
I'm using it on sway with wayland backend, everything is ok so far. It's ready to be reviewed.

File: srcpkgs/libdecor/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+build_options="dbus"
+build_options_default="dbus"
+
+post_install() {
+	vlicense LICENSE
+}
+
+libdecor-devel_package() {
+	depends="libdecor>=${version}_${revision}"
```
Comment (paper42) [inline srcpkgs/libdecor/template:25]:
```suggestion
	depends="${sourcepkg}>=${version}_${revision}"
```

**Review by paper42 — COMMENTED** [review]
lgtm, I have an issue with SDL2 with the Wayland backend being slow to register my keystrokes in CDDA, but it's not a regression and might not be a fault of SDL2.

**Comment by ardadem** [comment]
@paper42 If you want to test SDL2 more i can recommend https://github.com/ddnet/ddnet.

You need to set `DDNET_DRIVER=GLES` to use wayland backend with it.

**Pull request merged**